### ISSUE TITLE
about siglip2 and llama3.1-8b

### ISSUE THREAD

**Issue by mengcius** [microsoft/LLM2CLIP#51]
Great job. 
Could you elaborate on how "siglip2 and llama3.1-8b" is trained? 
Was the llm jointly optimized during the training of clip? 
Does the current code support the joint optimization of llm when training clip?

**Comment by raytrun** [comment]
The training method for "siglip2 and llama3.1-8b" is the same as for the other models mentioned in the paper. The primary difference is that we replaced the loss function with a sigmoid loss function. We also tried the original contrastive loss, which worked equally well, but ultimately chose the sigmoid loss to maintain consistency with SigLIP.

During the training of CLIP, the LLM parameters are frozen. Currently, the code doesn't support joint optimization of the LLM very well — the training speed still needs improvement.

**Comment by mengcius** [comment]
Thanks
